### DISCUSSION TITLE
Transfert of qMRIColors.jl to MagneticResonanceImaging organisation

### DISCUSSION THREAD

**Discussion by aTrotier** [MagneticResonanceImaging/MRIReco.jl#205] [General]
Hi all,

@mfuderer and I have developed the package [qMRIColors.jl](https://github.com/aTrotier/qMRIColors.jl) which is currently on my account. Can I move it to https://github.com/MagneticResonanceImaging ? @tknopp @JakobAsslaender @JeffFessler @alexjaffray @mrikasper 

Alternatively I can move it to https://github.com/JuliaHealth like KomaMRI.jl. Do you have some thoughts about that ? @cncastillo 
 
PS : Is it possible to open the discussion possibility at the organisation level ? @tknopp or somebody else who has the admin rights

**Comment by JeffFessler** [comment]
https://github.com/MagneticResonanceImaging makes sense to me.

**Comment by cncastillo** [comment]
Yeah, thumbs up for https://github.com/MagneticResonanceImaging 😁

**Comment by tknopp** [comment]
I agree. @aTrotier you have now sufficient access rights to do the move on your own.

**Reply by aTrotier** [reply]
Thanks
